cookbook/composer.rst

```diff
@@ -14,35 +14,24 @@ following sections.
 Install Composer on Linux and Mac OS X
 --------------------------------------
 
-To install Composer on Linux or Mac OS X, execute the following two commands:
+#. Run the installer as described in `the official Composer documentation`_.
```

Comment:
our standards indicate that every list item should end with a semi-colon (except from the last item).